.cicdtemplate/.bitrise/bitrise.yml

```diff
@@ -1,224 +1,283 @@
 ---
-format_version: '8'
-default_step_lib_source: 'https://github.com/bitrise-io/bitrise-steplib.git'
+format_version: '11'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 project_type: android
 trigger_map:
-  - push_branch: develop
-    workflow: develop
 
-  - push_branch: release/*
-    workflow: release
+- push_branch: develop
+  workflow: develop
 
-  - push_branch: master
-    workflow: master
+- push_branch: release/*
+  workflow: release
 
-  - push_branch: '*'
-    workflow: primary
+- push_branch: master
+  workflow: master
+  
+- push_branch: "*"
+  workflow: automatic_code_review
 
-  - pull_request_source_branch: '*'
-    workflow: primary
+- pull_request_source_branch: "*"
+  workflow: automatic_code_review
 
 # Work-flows definition
 workflows:
+
   # Workflow on * branch
-  primary:
+  automatic_code_review:
```

Comment:
I think we don't need to create a new workflow here since there is 1 more step added to run Danger. Could we apply Danger on primary workflow instead?

Reply:
it is not a new workflow, we can not create a new workflow with same triggers using Bitrise, I just rename it from `primary` to `automatic_code_review` for correctly reflecting the current workflow's functionalities 😅

Reply:
@minhnimble but the primary workflow is still [here](https://github.com/nimblehq/android-templates/pull/130/files#diff-9a67f94064187042e4635e3345dee64427399a151989b21b0eeb151616bacbc1R174). Imo, better to replace the primary with the new workflow or keep both of them (as you said it's impossible).

Reply:
@luongvo yeah I that is why I added the comment - `# Workflow not used for now` there as well to keep the `primary` workflow code in case we still need it later on 😅

Reply:
@minhnimble imo yes, let's ask the author too @hoangnguyen92dn ✋